framework/src/components/storage/entities/block.js

```diff
@@ -197,8 +197,8 @@ class Block extends BaseEntity {
 		});
 		this.addField('timestamp', 'number', { filter: filterType.NUMBER });
 		this.addField('totalAmount', 'string', { filter: filterType.NUMBER });
-		this.addField('totalFee', 'string', { filter: filterType.NUMBER });
-		this.addField('reward', 'string', { filter: filterType.NUMBER });
+		this.addField('totalFee', 'number', { filter: filterType.NUMBER });
+		this.addField('reward', 'number', { filter: filterType.NUMBER });
```

Comment:
So you just converted `totalFee` to string, why the change in genesis block schema caused to change the storage layer. Storage layer were already treating those as strings and it should not change.